.github/workflows/release-ios.yml

```
@@ -21,17 +22,15 @@ jobs:
           sparse-checkout-cone-mode: false
       - name: Get rusaint version
         id: current_version
-        uses: mikefarah/yq@v4
-        with:
-          cmd: yq '.workspace.package.version | "v" + .' Cargo.toml
+        run: yq '.workspace.package.version | "result=v" + .' Cargo.toml >> $GITHUB_OUTPUT
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:64: Double quote to prevent globbing and word splitting

(shellcheck)